examples/DropletCollision/templates/tpl_1_generateLiquid.xml

```diff
@@ -0,0 +1,160 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<mardyn version="20100525" >
+
+<refunits type="SI">
+	<length unit="nm">0.1</length>
+	<mass unit="u">1</mass>
+	<energy unit="K">1</energy>
+</refunits>
+
+<simulation type="MD" >
+	<integrator type="Leapfrog" >
+		<timestep unit="reduced" >0.003647347</timestep>
+	</integrator>
+
+	<run>
+		<currenttime>0</currenttime>
+		<production>
+			<steps>20000</steps>
+		</production>
+	</run>
+
+	<ensemble type="NVT">
+		<temperature unit="reduced" >ReplaceRedTemp</temperature>
+		<domain type="box">
+			<lx>ReplaceBox</lx>
+			<ly>ReplaceBox</ly>
+			<lz>ReplaceBox</lz>
+		</domain>
+
+		<components>
+			<include query="/components/moleculetype">../components.xml</include>
+		</components>
+
+		<phasespacepoint>
+			<generator name="CubicGridGenerator">
+				<specification>density</specification>
+				<density>ReplaceDensity</density>
+				<binaryMixture>false</binaryMixture>
+			</generator>
+		</phasespacepoint>
+
+	</ensemble>
+
+	<algorithm>
+		<parallelisation type="DomainDecomposition">
+			<!--<MPIGridDims> <x>2</x> <y>7</y> <z>2</z> </MPIGridDims>-->
+		</parallelisation>
+		<datastructure type="LinkedCells">
+			<cellsInCutoffRadius>1</cellsInCutoffRadius>
+		</datastructure>
+		<cutoffs type="CenterOfMass" >
+			<defaultCutoff unit="reduced" >2.5</defaultCutoff>
+			<radiusLJ unit="reduced" >2.5</radiusLJ>
+		</cutoffs>
+		<electrostatic type="ReactionField" >
+			<epsilon>1.0e+10</epsilon>
+		</electrostatic>
+		<thermostats>
+			<thermostat type="TemperatureControl">
+				<control>
+					<start>0</start>
+					<frequency>1</frequency>
+					<stop>1000000000</stop>
+				</control>
+				<regions>
+					<region>
+						<coords>
+							<lcx>0.0</lcx> <lcy>0.0</lcy> <lcz>0.0</lcz>
+							<ucx>box</ucx> <ucy>box</ucy> <ucz>box</ucz>
+						</coords>
+						<target>
+							<temperature>ReplaceRedTemp</temperature>
+							<component>0</component>
+						</target>
+						<settings>
+							<numslabs>5</numslabs>
+							<exponent>0.4</exponent>
+							<directions>xyz</directions>
+						</settings>
+						<writefreq>500</writefreq>
+						<fileprefix>temp_log</fileprefix>
+					</region>
+				</regions>
+			</thermostat>
+		</thermostats>
+	</algorithm>
+
+	<output>
+		<outputplugin name="CheckpointWriter">
+			<type>binary</type>
+			<writefrequency>20000</writefrequency>
+			<outputprefix>cp_binary_bulk</outputprefix>
+		</outputplugin>
+		<outputplugin name="MmpldWriter" type="multi">
+			<include query="/spheres">../sphereparams.xml</include>
+			<writecontrol>
+				<start>0</start>
+				<writefrequency>10000</writefrequency>
+				<stop>500000000</stop>
+				<framesperfile>0</framesperfile>
+			</writecontrol>
+			<outputprefix>megamol</outputprefix>
+		</outputplugin>
+		<outputplugin name="Adios2Writer">
+			<outputfile>adios_checkpoint.bp</outputfile>
+			<adios2enginetype>BP4</adios2enginetype>
+			<writefrequency>10000</writefrequency>
+		</outputplugin>
```

Comment:
Is this needed? I don't see any reference to these checkpoints anywhere else.
This also applies to the other simulation steps.

Reply:
I mainly use the `Adios2Writer` for visualization. It kind of doubles with the MMPLDWriter but the Adios2Writer contains also the velocities which makes it nice for some specific coloring in the visualization. I would rather get rid of the MMPLDWriter plugin.